app/components/admin/budgets/actions_component.rb

```diff
@@ -65,7 +65,17 @@ def ballots_action
                text: t("admin.budgets.actions.ballots"),
                path: create_budget_poll_path,
                method: :post,
-               confirm: t("admin.budgets.actions.confirm.ballots"))
+               confirm: t("admin.budgets.actions.confirm.ballots"),
+               disabled: !Setting["process.polls"])
```

Comment:
Perhaps here you can use the SettingsHelper method `feature?(name)` 🤔 
You can see this method in `app/helpers/settings_helper.rb:11`